docs/source/optimizations/qscout/qscout_qss.ipynb

```diff
@@ -30,111 +32,143 @@
    ]
   },
   {
+   "attachments": {},
    "cell_type": "markdown",
    "id": "8c861954",
    "metadata": {},
    "source": [
-    "## Imports and Token"
+    "## Imports and API Token\n",
+    "\n",
+    "For this tutorial, we will just need the imports listed below to run the Superstaq optimizations. If you do not have `qiskit-superstaq` installed on your machine, you can follow the guide [here](https://docs-superstaq.readthedocs.io/en/latest/get_started/installation.html#installing-qiskit-superstaq) to set it up and its dependencies. "
```

Comment:
i think we also talked about adding something like "this example notebook uses qiskit-superstaq; you can try it out by running pip install qiskit-superstaq[examples]" to the top of qiskit-superstaq notebooks